### DISCUSSION TITLE
Using on Mac

### DISCUSSION THREAD

**Discussion by ayoashy** [Cyfrin/security-and-auditing-full-course-s23#142] [Q&A]
After several attempts to install slither on my mac. I'm having challenge using the tool with solidity projects. I am getting tons of weird error messages after running the `slither .` command. Here is what the error message looks like

<img width="1186" alt="Screenshot 2024-03-13 at 4 51 42 PM" src="https://github.com/Cyfrin/security-and-auditing-full-course-s23/assets/100806750/d9737f36-1df2-4422-8593-f85bc7b0fff0">

**Comment by alymurtazamemon** [comment]
@ayoashy Your slither tool is working fine. It just gives these types of errors when it finds unsupported features in the smart contracts.

**Reply by ayoashy** [reply]
@alymurtazamemon please could you tell how to spot these unsupported features and possibly remove them cos I get this error in all solidity projects I cloned

**Reply by alymurtazamemon** [reply]
@ayoashy at the end, it tells about the error.
